Add a documentation outline

mjeronimo opened the pull request:
Add a structure for the Space ROS documentation site that has empty placeholder pages and TODO lists. Also update the logo and color scheme to better match the Space ROS website.

Commit:
Rough out an outline for SpaceROS docs

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Add more placeholder docs

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Integrate feedback from team review

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Add placeholder TODO items for all pages

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Remove some obsolete CSS

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Address linting issues

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

nuclearsandwich commented:
What do you think about consolidating some of the "Use $TOOL" articles into a single using static analysis tools page with sections? A lot of them are going to have the same start and end and only the middle will be different and we can avoid repeating a lot of content that way.

I'm also going back and forth in my mind about whether a lot of these how to guides need an explicit. "... with Space ROS" at the end of the title or if the Space ROS docs site has that implicitly. But part of me is irked by the idea of generic "how to use cobra" guides appearing on the docs site rather than being focused on the Space ROS integration. In particular I don't want to create a "Use Earthly" guide since they have tutorials for doing so I'd rather integrate that into a "First time building Space ROS" tutorial and "Reproducing CI results locally" guide respectively.

Commit:
Incorporate feedback on a couple CI-related pages; also add a bit of intro text

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

Commit:
Incorporate feedback on a couple CI-related pages; also add a bit of intro text

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

mjeronimo commented:
@nuclearsandwich  I don't think we have enough pages yet on static analyzers to consolidate. Open to doing that or creating a sub-tree for static analysis when we have more.

I think we can be careful with the "... with Space ROS" items. Only use it if helps to avoid ambiguity, for example. 

I've added the suggested CI-related pages and dropped the "Using Earthly" page.

nuclearsandwich reviewed (approved):
Thanks for considering the review feedback. I think we can take this as is and then adjust as we start filling out the articles.
